newdbimport

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+function newdbimport() {
```

Comment:
so, we already have wpdbimport doing this and more. when we start getting a bunch of functions that do the same thing, things get confusing. a better approach would be to modify wpdbimport so you can pass in a flag, or have it detect if it should try to edit a config file. it would be cool if it got turned into a command like 
"db import" and it will detect and update configs for joomla, wordpress, drupal, moodle, etc.

I would prefer that this file is removed from this pull request and that wpdbimport is modified. just make sure that when you do modify wpdbimport that it doesnt break other functions that may call it

Reply:
will do in next pull request